DEVELOPMENT.md

```diff
@@ -164,8 +164,10 @@ The Tekton project requires that you develop (commit) code changes to branches t
 
         ```shell
         git remote add upstream git@github.com:tektoncd/pipeline.git
+        ```
 
-        # Optional: Prevent accidental pushing of commits by changing the upstream URL to `no_push`
+    3. Optional: Prevent accidental pushing of commits by changing the upstream URL to `no_push`
+        ```shell
```

Comment:
Why need this change

Reply:
Thanks for review
Prior , it is as below:
![Screenshot 2023-10-05 at 11 23 52 PM](https://github.com/tektoncd/pipeline/assets/25217700/2954a763-8d97-4a53-8fe1-1cec3caa47d3)

When copied and pasted in zsh shell , below error is thrown.
```
# Optional: Prevent accidental pushing of commits by changing the upstream URL to `no_push`
git remote set-url --push upstream no_push
zsh: command not found: no_push
zsh: command not found: #
```

When optional section is moved to new line as below, the issue is not seen
![Screenshot 2023-10-05 at 11 26 08 PM](https://github.com/tektoncd/pipeline/assets/25217700/ef686f63-d880-4f8d-b9ab-7e2c4b41e16f)